Assets/Scripts/Utilities/RenderSettingsManager.cs

```diff
@@ -46,6 +44,15 @@ void Awake()
 
 	void Update()
 	{
+		dayCycleTimer += Time.deltaTime;
+
+		timeOfDay = (-Mathf.Cos(dayCycleTimer/dayCycleTime * Mathf.Rad2Deg) * 0.5f) + 0.5f;
```

Comment:
Ok. From my understanding it would make sense to put 2Pi there. Maybe it's a scaling issue with the day length? ie the day length as is doesn't really correspond to a real value like seconds because it is being scaled by that deg2rad.